sos/report/plugins/targetcli.py

```diff
@@ -23,9 +23,14 @@ def setup(self):
             "targetcli ls",
             "targetcli status",
         ])
+        sys_conf_dir = '/sys/kernel/config/target'
+        self.add_forbidden_path([
+            self.path_join(sys_conf_dir, 'iscsi/discovery_auth/password*'),
+            self.path_join(sys_conf_dir, '*/*/auth/password*'),
+        ])
```

Comment:
`add_forbidden_path()` calls `fnmatch.translate()` internally, which I believe should be able to handle `**` like `glob` does, meaning we can do `self.path_join(sys_conf_dir, '**/password*') to get any future and/or unknown password file paths.